react/conclusion/conclusion.md

```diff
@@ -12,6 +12,8 @@ Furthermore, you can dive into React meta frameworks. They're a framework for Re
 
 You're at a point in your React journey where you can begin learning about design patterns and architecture. [patterns.dev](https://www.patterns.dev/) is an exceptional resource that will help you build better React apps by leveraging powerful patterns. They are worth a bookmark!
 
+Before you move onto the next section. Fill out this survey [here](https://docs.google.com/forms/d/e/1FAIpQLSdj_tNMp0LEz3ZLPqYcF67V11tX_CCJP3CTictPZzZ6XQm2Gw/viewform?usp=sf_link) to add your input and experience with the section. This helps us improve the section as well as the overall course and is greatly appreciated.
```

Comment:
We could have better link text here:
```suggestion
Before you move onto the next section. Fill out this [React course feedback survey](https://docs.google.com/forms/d/e/1FAIpQLSdj_tNMp0LEz3ZLPqYcF67V11tX_CCJP3CTictPZzZ6XQm2Gw/viewform?usp=sf_link) to add your input and experience with the section. This helps us improve the section as well as the overall course and is greatly appreciated.
```

Reply:
@01zulfi Sure. Main reason I did that is in case we ever add another path we don't have to go back and change the survey. I'm down with radio buttons if you think they fit better although.